main/test/flix/Test.Dec.Class.flix

```diff
@@ -145,11 +145,11 @@ namespace Test/Dec/Class {
         lawless class C[a] {
             pub def f(): a
 
-            law l(): Bool = true
+            law l: forall (_x: a) . true
 
             pub def g(): a
 
-            law m(): Bool = true
+            law m: forall[a](_x: a) . true
```

Comment:
Does this [a] shadow the a of the class?

Reply:
I think it is actually bound to the class's a